src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariants.java

```diff
@@ -806,29 +781,6 @@ private SampleDB initializeSampleDB() {
     }
 
     /**
-     * Get IDs from a file
-     *
-     * @param file file containing the IDs
-     * @return set of IDs or null if the file is null
-     * @throws UserException.CouldNotReadInputFile if could not read the file
-     */
-    private Set<String> getIDsFromFile(final File file){
-        /** load in the IDs file to a hashset for matching */
-        if (file != null) {
-            Set<String> ids = new LinkedHashSet<>();
-            try (final XReadLines xrl = new XReadLines(file)) {
-                ids.addAll(xrl.readLines().stream().map(String::trim).collect(Collectors.toList()));
```

Comment:
Will barclay also do the `String.trim()` operation when populating a Collection argument from a file?